src/index.html

```diff
@@ -17,16 +17,30 @@
     />
   </head>
   <body>
-    <input
-      type="text"
-      data-qa="keypress"
-      placeholder="Try “Los Angeles“"
-    />
+    <div
+      class="container"
+      data-qa="big"
```

Comment:
The `data-qa` attribute for the big search form is incorrectly assigned as `big`. Please ensure it matches the task requirements.